StitchCore/StitchCore/Source/Core/StitchHTTPClient.swift

```diff
@@ -102,6 +106,8 @@ internal class StitchHTTPClient {
         try deleteToken(withKey: Consts.AuthRefreshTokenKey)
         try deleteToken(withKey: Consts.IsLoggedInUDKey)
```

Comment:
I think we should be doing `userDefaults?.removeObject(forKey: Consts.IsLoggedInUDKey)` here because with `deleteToken` it will try to delete the key from the keychain when not using the simulator. Should I make this drive-by change or is this here intentionally?